media/livestream/livestream_test.go

```diff
@@ -517,6 +525,64 @@ func TestChannelEvents(t *testing.T) {
 		}
 	})
 
+	// Tests for clips
+	// Send a test stream for the clip.
+	cmd := exec.Command("ffmpeg", "-re", "-f", "lavfi", "-t", "45", "-i",
+		"testsrc=size=1280x720 [out0]; sine=frequency=500 [out1]", "-vcodec",
+		"h264", "-acodec", "aac", "-f", "flv", uri)
+
+	_, err := cmd.CombinedOutput()
+	if err != nil {
+		t.Fatalf("exec.Command: %v", err)
+	}
+
+	// Create a new channel clip.
+	clipOutputUri := fmt.Sprintf("%sclips", outputURI)
+	testutil.Retry(t, 3, 2*time.Second, func(r *testutil.R) {
+		clipName := fmt.Sprintf("projects/%s/locations/%s/channels/%s/clips/%s", tc.ProjectID, location, channelID, clipID)
+		if err := createChannelClip(buf, tc.ProjectID, channelID, clipID, clipOutputUri); err != nil {
+			r.Errorf("createChannelClip got err: %v", err)
+		}
+		if got := buf.String(); !strings.Contains(got, clipName) {
+			r.Errorf("createChannelClip got\n----\n%v\n----\nWant to contain:\n----\n%v\n----\n", got, clipName)
```

Comment:
nit: I'd prefer to not have the test output formatted in this way. It makes it harder to read on GitHub.

The test input can be as simple as:
```go
r.Errorf("createChannelClip: want: %v, got %v", clipName, got)
```